frontend/src/index.js

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+import './index.scss';
+import Navbar from './components/Navbar';
+import MealSection from './components/MealSection';
+import Header from './components/Header'
+import SearchBar from './components/SearchBar'
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    {/* TODO: Search Component */}
+    <SearchBar />
+
+    {/* TODO: Header Component */}
+    <Header locationName='USC Village Dining Hall' />
+    {/* TODO: MealSection Component */}
+        {/* TODO: MealItem Component */}
+    <MealSection stationName='Plant Based' />
+    <MealSection stationName='Flexiterian' />
+    {/* TODO: Navbar Component */}
+    <Navbar />
   </React.StrictMode>,
   document.getElementById('root')
 );
```

Comment:
I'd like most of this to live in a separate <Menu/> component, but I think @concretelinguist will be building this with the server fetching logic, so we can wait until after she's done to move it over.